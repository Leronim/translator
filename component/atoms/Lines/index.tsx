import React from 'react';
import { View } from 'react-native';

const Lines = () => {
    const array = new Array(3).fill('')
    return (
        <View style={{ height: 70 }}>
            {array.map((_: undefined, index: number) => {
                return (
                    <View 
                        key={index} 
                        style={{ 
                            width: '100%',
                            height: 1,
                            backgroundColor: '#4B4B4B',
                            top: index * 30 - 2,
                         }}
                    />
                )
            })}
        </View>
    )
}

export default Lines