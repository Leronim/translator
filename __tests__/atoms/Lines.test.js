import React from 'react';
import Lines from '../../component/atoms/Lines';
import { render } from '@testing-library/react-native';

test('Lines render', () => {
    const tree = render(<Lines/>).toJSON();
    expect(tree).toMatchSnapshot();
})