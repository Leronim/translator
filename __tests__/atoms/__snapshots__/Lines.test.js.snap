// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Lines render 1`] = `
<View
  style={
    Object {
      "height": 70,
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "#4B4B4B",
        "height": 1,
        "top": 0,
        "width": "100%",
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "#4B4B4B",
        "height": 1,
        "top": 55,
        "width": "100%",
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "#4B4B4B",
        "height": 1,
        "top": 110,
        "width": "100%",
      }
    }
  />
</View>
`;
